src/Form/Form.php

```diff
@@ -546,4 +535,118 @@ private function getHiddenFields(array $items): array
 
 		return $output;
 	}
+
+	/**
+	 * Get secure form data.
+	 *
+	 * @param string $prefix Prefix of the form.
+	 * @param array<string, mixed> $attributes Attributes of the form.
+	 *
+	 * @return string
+	 */
+	private function getSecureFormData(string $prefix, array $attributes): string
+	{
+		$output = [];
+
+		$parentSettings = $attributes["{$prefix}ParentSettings"] ?? [];
+		if (!$parentSettings) {
+			return '';
+		}
+
+		// Output variation.
+		$variation = $parentSettings['variation'] ?? [];
+		if ($variation) {
+			$variationOutput = [];
+			foreach ($variation as $value) {
+				$title = $value['title'] ?? '';
+				$slug = $value['slug'] ?? '';
+
+				if (!$title || !$slug) {
+					continue;
+				}
+
+				$variationOutput[] = [
+					$title,
+					$slug,
+				];
+			}
+
+			$output['v'] = $variationOutput;
+		}
+
+		// Legacy attributes - this will be removed in the future.
```

Comment:
A VS Code extension recommendation incoming
Assuming there are several instances of these you could utilize a TODO plugin to help with these kind of tasks, this one is the most popular: https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree

Reply:
tnx but this is going out once Infobip implements the changes so I'm ok with the current way. tnx for this suggestion